.coderabbit.yaml

```
@@ -0,0 +1,65 @@
+  review_status: true
+  collapse_walkthrough: true # 긴 리뷰 요약을 접어서 깔끔하게 표시
+  auto_review:
+    enabled: true
+    drafts: false
+    base_branches:
+      - "main"
+      - "develop"
+      - "dev"
+      - "release"
+chat:
+  auto_reply: true
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)